### PR TITLE
create a small PR for testing

### PR DESCRIPTION
**Pull request by ScientificCodingKids**

### CODE REVIEW

File: V5/SpinUp/src/main.cpp [hunk 1]
```diff
@@ -10,6 +10,10 @@ using namespace std;
 
 competition Competition;
 
+double multby2(double x) {
+  return x + 2.0;
```
Comment (chloen13) [inline V5/SpinUp/src/main.cpp]: return x * 2.0;

**Review by chloen13 — COMMENTED** [review]
* not +

File: V5/SpinUp/src/main.cpp (hunk 1 above)
Comment (ScientificCodingKids) [reply V5/SpinUp/src/main.cpp]: nice catch!

**Review by chloen13 — APPROVED** [review]
yes

**Pull request merged**